### PR TITLE
 upgrade to origami version 2.0

### PR DESCRIPTION
**Pull request by JakeChampion**
This was done following the guide at https://gist.github.com/notlee/8338b043e26ae3cd9c1f0878e1e6ce7c

### CODE REVIEW

File: origami.json [hunk 1]
```diff
@@ -1,9 +1,7 @@
 {
-	"description": "The default colour palette for all FT products. The palette supports colour contrast checking, colour mixing and toneing.",
-	"origamiType": "module",
-	"origamiCategory": "primitives",
-	"keywords": "o-colours, colours, themes, schemes, pink, salmon, slate, tone, mix, background, palette",
-	"origamiVersion": 1,
+	"origamiType": "component",
+	"type": "module",
```
Comment (chee) [inline origami.json:3]:
```suggestion
```

File: .github/workflows/test-origami-component.yml [hunk 2]
```diff
@@ -5,18 +5,11 @@ jobs:
...
-      id: repo_name
-      run: echo ::set-output name=REPO_NAME::${GITHUB_REPOSITORY/*\//}
-    - run: npx origami-ci branch
-      env:
-        ORIGAMI_CI_NAME: ${{ steps.repo_name.outputs.REPO_NAME }}
-        ORIGAMI_CI_VERSION: 0.0.0
-        GITHUB_TOKEN: ${{ secrets.ORIGAMI_FOX_ACCESS_TOKEN }}
-        ORIGAMI_PULL_REQUEST: ${{ github.event.pull_request.html_url }}
+    - run: npm install -g npm@^7
+    - run: npm install -g origami-build-tools@prerelease
```
Comment (chee) [inline .github/workflows/test-origami-component.yml:12]:
```suggestion
    - run: npm install -g origami-build-tools@2
```

File: package.json [hunk 3]
```diff
@@ -1,18 +1,48 @@
...
+		"npm": "^7"
+	},
 	"devDependencies": {
 		"eslint": "^7.8.1",
 		"eslint-config-origami-component": "^2.1.0",
-		"origami-ci-tools": "^2.0.2",
 		"remark": "^13.0.0",
 		"remark-lint": "^8.0.0",
-		"remark-preset-lint-origami-component": "^1.2.0",
+		"remark-preset-lint-origami-component": "prerelease",
```
Comment (chee) [inline package.json:32]:
```suggestion
		"remark-preset-lint-origami-component": "^2.0.0",
```

and package-lock.json will be updated

**Comment by chee** [comment]
woops. sorry, frank

**Review by chee — APPROVED** [review]
***

**Comment by JakeChampion** [comment]
I can't review as I opened the pr, but I do approve 🙂

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v6.0.0 :tada:

**Pull request merged**